### PR TITLE
docs(aio): add blox material library to resources

### PR DESCRIPTION
**Pull request by gjdev**
Adds the Blox Material library to the list of resources. Blox Material integrates Material Components for the Web with Angular.

```
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```
## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by StephenFluin — APPROVED** [review]

**Comment by gjdev** [comment]
Is this still considered to be merged? This pull request was approved over 6 months ago (by @StephenFluin ). But since then nothing has changed. Is there maybe some action I still need to take? Could somebody from the angular team please let me know how to either get this PR merged, or get on update on what the status of this proposal is? Many thanks.

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -364,6 +364,13 @@
             "url": "http://www.amexio.tech/",
             "logo": "http://www.amexio.org/amexio-logo.png"
           },
+          "bm": {
+            "desc": "A lightweight Material Design library for Angular, based upon Google's Material Components for the Web",
+            "logo": "",
```
Comment (gkalpak) [inline aio/content/marketing/resources.json]: No logo?

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
Sorry, @gjdev. Seems like this fell through the cracks. Can you please squash the commits into one?
<sub>(Once you've done that, ping me and I will mark it for merging.)</sub>

**Comment by gjdev** [comment]
@gkalpak thanks for that! It's now a single commit, and I've added a logo as suggested by you.

The circleci status now displays some test failures, but that doesn't look to be related to this PR.

Thanks again!

**Comment by gkalpak** [comment]
LGTM (the CI failures couldn't have been caused by this PR).

**Pull request closed without merging**